src/components/StyledButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from 'expo-router';
+import React from 'react';
+import { Pressable, Text, TouchableOpacity } from 'react-native';
+import styles from './styles';
+
+function StyledButton({ text, file }: { text: string; file: string }) {
```

Comment:
Add functionality so that if `file` is null, only return the `TouchableOpacity`, otherwise use the `Link` component

Reply:
Let me know if more explanation would be helpful btw!